app/helpers/remote_execution_helper.rb

```diff
@@ -99,7 +107,6 @@ def job_invocation_task_buttons(task)
     job_invocation = task.task_groups.find { |group| group.class == JobInvocationTaskGroup }.job_invocation
     task_authorizer = Authorizer.new(User.current, :collection => [task])
     buttons = []
-    buttons << link_to(_('Refresh'), {}, :class => 'btn btn-default', :title => _('Refresh this page'))
```

Comment:
I removed this as I did not see a reason for it to exist anymore - page elements are updated.